src/upcoming-components/Drawer/Drawer.tsx

```diff
@@ -216,6 +215,8 @@ export const Drawer = ({
           onClick={handleOnClickOutside}
           onKeyDown={handleOnKeyDown}
           ref={ref}
+          role="button"
+          tabIndex={0}
           {...other}
```

Comment:
error: 
1.Static HTML elements with event handlers require a role.
2.Elements with the 'button' interactive role must be tabbable.
solution:
Added role and tabIndex